actions/publish-github-pages/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine:3.13
+#FROM debian:stretch
+#
+#LABEL maintainer="Cloud Posse <hello@cloudposse.com>"
+#
+#LABEL "com.github.actions.name"="Publish Hugo to GitHub Pages"
+#LABEL "com.github.actions.description"="Build a Hugo website and publish it to GitHub Actions."
+#LABEL "com.github.actions.icon"="activity"
+#LABEL "com.github.actions.color"="blue"
+#
+#RUN	apk add --no-cache \
+#	bash \
+#	ca-certificates \
+#	curl \
+#	jq \
+#	git
```

Comment:
Why is all this commented out?

Reply:
Great question. I did that back when I was debugging the GitHub Action (trying to get the `uses: composite` run type to work and ultimately giving up). I've cleaned up it now, but it's interesting that the action still worked with all that commented out.